### DISCUSSION TITLE
How can I help?

### DISCUSSION THREAD

**Discussion by QgitboltQ** [FreeTubeApp/FreeTube#3732] [General]
hi all, 
I am a 3ish-year gnu/linux CLI hobbyist with who an interest in system administration, privacy and security.
also have some mild python knowledge (I haven't used python in about 2 years) and I could spruce up if needed. 
how can I help? 
could clean up some documentation or do some other technical or non-technical house-cleaning tasks, preferably gnu/linux-related.
I noticed that @GilgusMaximus mentioned in his [latest blog post ](https://blog.freetubeapp.io/behind-the-scenes-improving-project-structure-and-other-updates?pk_campaign=rss-feed) that it is preferable to have a Discussion rather than email freetubeapp@protonmail.com so here I am!
I donate via liberapay but it would be nice to be more involved as this feels like the right project for me. 
feel free to reply with any ideas.
cheers to this sweet project.

**Comment by PrestonN** [comment]
Hello! I really appreciate you wanting to help out with the project. We are always more than happy to have more potential contributors.

Development wise, we use a combination of Electron / Vue.js for our tech stack, so familiarity in those would be a plus. You can check out our [Getting Started](https://docs.freetubeapp.io/development/getting-started/) guide with more information on how to setup a local environment as well as some useful documentation to learn out tech stack a bit better. Once you have everything up and running, feel free to take a look at our [Good First Issue](https://github.com/FreeTubeApp/FreeTube/labels/good%20first%20issue) list and see if anything looks appealing that you'd like to try and work on. Once finished, we will review your code and put into the main repository once satisfactory.

If you're not comfortable with doing any development, then there are still ways that you can help. If you happen to know multiple languages, you can check out the project over on [Weblate](https://hosted.weblate.org/engage/free-tube/) and help contribute translations for the project. If we already support that language, you can still proofread the translations that we have and make corrections as needed.

Another great way to help is to simply be active here on GitHub. Creating issues, testing pull requests, testing nightly builds, and helping resolve existing issues is always very helpful. Being able to resolve these quicker allows us to focus on other aspects of the app sooner.

Finally, you mentioned documentation. As linked above, we do have a [Docs Site](https://docs.freetubeapp.io/) along with a [repo](https://github.com/FreeTubeApp/FreeTube-Docs) to go with it. Feel free to look through there and see if there are any changes / additions you'd like to see. You're also welcome to make drafts yourself and we can review them as they're made.

I'm looking forward to seeing any contributions from you. Feel free to let us know if you happen to have any questions about anything. We are also very active over on [Matrix](https://matrix.to/#/+freetube:matrix.org) where you might get a faster response from someone.
